de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/MWindow.java

```diff
@@ -53,7 +54,7 @@ public MWindow (Properties ctx, int AD_Window_ID, String trxName)
 		super (ctx, AD_Window_ID, trxName);
 		if (AD_Window_ID == 0)
 		{
-			setWindowType (WINDOWTYPE_Maintain);	// M
+			setWindowType (WINDOWTYPE_Verwalten);	// M
```

Comment:
Instead of changing to german names, pls fix the AD_Ref_List.ValueName for WINDOWTYPE list.